runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerLog.java

```diff
@@ -40,13 +43,29 @@
 import net.runelite.client.util.StackFormatter;
 
 @Getter
-class LootTrackerBox extends JPanel
+class LootTrackerLog extends JPanel
```

Comment:
LootTrackerBox was better name as it indicated more that it was component, this is more confusing. Use the original name